packages/vue-client/vite.config.ts

```diff
@@ -9,6 +9,12 @@ export default defineConfig({
   resolve: {
     alias: {
       "@": fileURLToPath(new URL("./src", import.meta.url)),
+      "@ogfcommunity/variants-shared/src": fileURLToPath(
+        new URL("../shared/src", import.meta.url),
```

Comment:
`@ogfcommunity/variants-shared/src` is to support imports like "@ogfcommunity/variants-shared/src/time_control"

`@ogfcommunity/variants-shared/src` is to support imports like "@ogfcommunity/variants-shared".

Importantly, the /src version needs to come first, else the latter will overtake it.